README.md

```diff
@@ -1,116 +1,248 @@
+
+## Extracting translatable strings from the Source
+
+The `.po` files in the Locales don't have to be set manually: they can be generated.  
+The way to generate them is the following:
+
+### Command for translatable strings extraction:
+
+The following comman extracts all translatable strings from the **Source**:
 
 ```
 make gettext
 ```
 
-This will generate files located in `build/gettext/` with translatable 
-strings extracted from the `.rst` files. 
-Now generate the translation files with the following command:
+It generates a new directory in the `build` directory: `build/getext` 
+The directory tree matches that of the `source/locale` directory trees.
+So for every `.rst` file in the **Source** it contains a `.pot` file 
+
+### Identify the translatable String for your Source change
+
+Find the `.pot` file that contains the translatable strings. 
+
+In the example above this would be `build/getext/index.pot`
 
 ```
-sphinx-intl update -p build/gettext -l de -l fr
+#: ../../source/index.rst:15
+msgid "Schön, dass Sie Open Government Data publizieren möchten! Wir unterstützen Sie dabei."
+msgstr ""
 ```
 
-The generated files will be located in `source/locale/fr/LC_MESSAGES` and 
-`source/locale/de/LC_MESSAGES`. Only the files in 
-`source/locale/fr/LC_MESSAGES` need further work: 
-Fill in the translations in the `.po` files. Once you are ready commit your 
-updated `.po` files to github.
+### Transfer the translatable string to the Locale
 
-To test the translations do the following to
-compile `.mo` files and build the translated html files for both languages:
+Copy thins into the `locale/fr/LC_MESSAGES/index.po` and replace the old phrase there by this 
+new phrase: 
 
 ```
-make -e SPHINXOPTS="-D language='fr'" BUILDDIR="build/fr" html
-make -e SPHINXOPTS="-D language='de'" BUILDDIR="build/de" html
- ```
+#: ../../source/index.rst:15
+msgid ""
+"Gut, dass Sie Open Government Data publizieren möchten! Wir "
+"unterstützen Sie dabei."
+msgstr ""
+"Vous souhaitez publier des données publiques ouvertes? Nous vous "
+"apportons notre soutien!"
+```
 
-See the [sphinx-intl documentation](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html)
-for more details. Load the index file in to the browser as described above 
-in the section on the German version.
+Now you would be again ready to build the French Locale.
 
-## Linkchecker
+## Command to transfer the tranlatable strings automatically
 
-The documentation includes a check of its external links.
+The translatable string can be automatically transfered from the `.pot`files to the 
+Locales. 
 
-You can run it with make:
+```
+sphinx-intl update -p build/gettext -l de -l fr
+```
+
+:WARNING: Use this command with care as it has the following pitfalls:
+
+- some translations are marked with `#, fuzzy` in the Locales by the command and this command has to be removed, so that the translations will work
```

Comment:
```suggestion
- some translations are marked with `#, fuzzy` in the Locales by the command and this line has to be removed, so that the translations will work
```